spec/stress_spec.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
-RSpec.describe "stress test", zookeeper: true, proxy: true, stress: true do
-  let(:logger) { Logger.new(ENV["ZK_RECIPES_DEBUG"] ? STDERR : nil) }
+RSpec.describe "stress test", zookeeper: true, proxy: true, slow: true do
+  let(:logger) { Logger.new(ENV["ZK_RECIPES_DEBUG"].present? ? STDERR : nil) }
```

Comment:
Just a nit, but if `ENV["ZK_RECIPES_DEBUG"] = "0"` it'll still be present, `"0".present? #=> true`.

I think it might be more straightforward logic to just check if the env var is set, `STDERR if ENV.key?("ZK_RECIPES_DEBUG")`. It doesn't handle the empty String case, but it almost seems better to simply rely on the var being set or not if not following env var truthy conventions.

Just an aside, but `ActiveModel::Type::Boolean.new.cast("0") #=> false`.

Reply:
Ya, I've gone back and forth on this. Gonna leave as is since it is only in tests.